cloudinit/net/sysconfig.py

```diff
@@ -444,10 +461,14 @@ def _render_subnets(cls, iface_cfg, subnets, has_default_route):
     @classmethod
     def _render_subnet_routes(cls, iface_cfg, route_cfg, subnets):
         for _, subnet in enumerate(subnets, start=len(iface_cfg.children)):
+            subnet_type = subnet.get('type')
             for route in subnet.get('routes', []):
                 is_ipv6 = subnet.get('ipv6') or is_ipv6_addr(route['gateway'])
```

Comment:
is_ipv6 is true if route['gateway'] is an ipv6 address here.

Reply:
Ah, perfect.  Thanks!